### ISSUE TITLE
zeroUnits false negative for unitless values

### ISSUE THREAD

**Issue by mehcode** [rossPatton/stylint#165]
```stylus
html, body {
  font-weight: 500;
}
```

```
Warning: Including the unit value is preferred
File: [...]
Line: 2: font-weight: 500;

Stylint: 0 Errors.
Stylint: 1 Warnings.
```

---

```
$ stylint --version
Stylint version: 1.0.9
```

---

On another note -- a lint to enforce a numerical font-weight would be nice.

**Comment by rossPatton** [comment]
this should be good in 1.0.10
